types/ChatMessage.ts

```diff
@@ -0,0 +1,3 @@
+export default interface ChatMessage {
+  Message: string;
+}
```

Comment:
Heeft een `ChatMessage` alleen een message string? Hoe weten we van welke kant het bericht is?